clients/ui/bff/docs/dev-guide.md

```diff
@@ -0,0 +1,82 @@
+# Development Guide
+
+## Local kubernetes deployment of Model Registry
+To test the BFF locally without mocking the k8s calls the Model Regsitry backend can be deployed locally using kind.
+
+### Prerequisites 
+The following tools need to be installed in your local environment:
+* Podman (Docker should also work) - [Podman Desktop Instructions](https://podman-desktop.io)
+* kubectl - [Instructions](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* kind - [Instructions](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+
+Note: all of the above tools can be installed using your OS package manager, this is the preferred method.
+
+### Setup
+#### 1. Create a kind cluster
+Create a local cluster for running the MR backend using the following command:
+```shell
+kind create cluster --name model-registry
```

Comment:
wdyt about not specifying the kind cluster name, so you don't have to repeat `--context kind-model-registry` in every kubectl command ?